app/pages/notifications/notification-section.jsx

```diff
@@ -117,6 +118,15 @@ export default class NotificationSection extends Component {
     });
   }
 
+  markAllRead() {
+    talkClient.put('/notifications/read', { section: this.props.section });
+    this.state.notificationData.forEach((data) => {
+      data.notification.update({ delivered: true }).save();
```

Comment:
You don't want to call `.save()` on these -- the update is already done in the put request